### ISSUE TITLE
[SUGGESTION] | Give the Dragon aspects a big Daddy

### ISSUE THREAD

**Issue by Sielentus** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth#326]
**DO NOT REMOVE PRE-EXISTING LINES**
----------------------------------------------------------------------------------------------------------
**Describe your suggestion in full detail below:**
Galakrond is technically the reason why the 5 Dragon aspects exist, because they defeated him and got the Power from the Titans and according to
https://wowwiki.fandom.com/wiki/Galakrond they are also Relative with our big boy

**Comment by zumbak04** [comment]
> https://wowwiki.fandom.com/wiki/Galakrond they are also Relative with our big boy, so i think he should be their big dead Daddy

He's not their daddy. Only Ysera and Alexstrasza are sisters and they were born in the orange proto-dragonflight.

**Comment by Sielentus** [comment]
> He's not their daddy. Only Ysera and Alexstrasza are sisters and they were born in the orange proto-dragonflight.
oh ok, i did not knew. i thought, Ysera and Alexstrasza  was based on the green and red proto dragons, alone because of the art picture 
![Wow dawn aspects galakrond 580](https://user-images.githubusercontent.com/49923433/57141937-acfbb600-6dbb-11e9-97f7-b185b9ba362c.jpg)

**Comment by zumbak04** [comment]
> oh ok, i did not knew. i thought, Ysera and Alexstrasza was based on the green and red proto dragons, alone because of the art picture ( and theire color)

As I know, it's fan art and if the red dragon is Alexstrasza, she should be orange, not red.

> oh yes a topic beside this, was the wonder and idea list so terrible? :-:

Do not worry. They're gud, but the implementation of some of them may take a lot of time. I'll leave comments there when I add something.

**Comment by Sielentus** [comment]
>As I know, it's fan art and if the red dragon is Alexstrasza, she should be orange, not red.

indeed, mh sorry then i have implanted too much thought the picture. To be fair, i got never the time to read  Dawn of the Aspects part: 1 (if it is mentioned in it) :3 but even if Galakrond is not necessarily closely related to the dragons at all, though he was indirectly responsible for the form and intelligence they have now. he need a bit love (: 

> Do not worry. They're gud, but the implementation of some of them may take a lot of time. I'll leave comments there when I add something.

that is nice to hear. i have spend a whole day on both list ( on https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/issues/325 mostly) i tried to make it lore friendly as possible ( and try to keep in mind, if its even possible to make) and to take some wonders that could be consider on these races/factions  as "wonders". for example nectroic and Scarlet light, was pretty hard to find something. In the last couple days i found more "wonder" and  Event ideas, that could be inplented someday. ( for example: Uthers Tomb when i die as wonder and a Event chain for Light and Blood elf).
Better i wait, untill  https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/issues/325  and https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/issues/327 make a cut into: Developer Suggestions in Suggestion Board (:
